Move quarkus_test and spring_test from extended to sanity level

Currently the [quarkus_test](https://github.com/adoptium/aqa-tests/blob/master/external/quarkus/playlist.xml#L16-L28) and [spring_test](https://github.com/adoptium/aqa-tests/blob/master/external/spring/playlist.xml#L16-L28) targets are in extended level, and since we are organizing frameworks into sanity and applications into extended, it will be better to change the level of these two targets to sanity (which means they will be run more regularly).

Changing these lines so that they are sanity instead of extended:
- https://github.com/adoptium/aqa-tests/blob/master/external/quarkus/playlist.xml#L23 
- https://github.com/adoptium/aqa-tests/blob/master/external/spring/playlist.xml#L23

will accomplish the task.

Just for the record, quarkus_test and spring_test failed with both openj9 and hotspot. The errors are the same for both SDK.

**JDK11 openj9 in Grinder/22106:**

- spring_test:
```
[2022-03-29T16:28:40.871Z] =JAVA VERSION OUTPUT BEGIN=
[2022-03-29T16:28:41.775Z] openjdk version "11.0.15-internal" 2022-04-19
[2022-03-29T16:28:41.775Z] OpenJDK Runtime Environment (build 11.0.15-internal+0-adhoc..BuildJDK11x86-64linuxNightly)
[2022-03-29T16:28:41.775Z] Eclipse OpenJ9 VM (build master-2e63c71, JRE 11 Linux amd64-64-Bit Compressed References 20220329_255 (JIT enabled, AOT enabled)
[2022-03-29T16:28:41.775Z] OpenJ9   - 2e63c71
[2022-03-29T16:28:41.775Z] OMR      - 796ccdf
[2022-03-29T16:28:41.775Z] JCL      - 43e18bf based on jdk-11.0.15+7)
[2022-03-29T16:28:41.775Z] =JAVA VERSION OUTPUT END=
[2022-03-29T16:28:41.775Z] Compile and execute spring-test
[2022-03-29T16:28:41.775Z] Building Spring  using gradle
[2022-03-29T16:31:25.181Z] Compilation with Kotlin compile daemon was not successful
[2022-03-29T16:31:25.181Z] java.lang.NullPointerException
[2022-03-29T16:31:25.181Z] 	at org.jetbrains.kotlin.incremental.IncrementalModuleEntry.hashCode(IncrementalModuleInfo.kt)
[2022-03-29T16:31:25.181Z] 	at java.base/java.util.HashMap.hash(HashMap.java:340)
[2022-03-29T16:31:25.181Z] 	at java.base/java.util.HashMap.put(HashMap.java:608)
[2022-03-29T16:31:25.181Z] 	at java.base/java.util.HashSet.readObject(HashSet.java:343)
[2022-03-29T16:31:25.181Z] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[2022-03-29T16:31:25.181Z] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[2022-03-29T16:31:25.181Z] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2022-03-29T16:31:25.181Z] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[2022-03-29T16:31:25.181Z] 	at java.base/java.io.ObjectStreamClass.invokeReadObject(ObjectStreamClass.java:1175)
[2022-03-29T16:31:25.181Z] 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2478)
[2022-03-29T16:31:25.181Z] 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2344)
[2022-03-29T16:31:25.181Z] 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1803)
[2022-03-29T16:31:25.181Z] 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:577)
[2022-03-29T16:31:25.181Z] 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:487)
[2022-03-29T16:31:25.181Z] 	at java.base/java.util.HashMap.readObject(HashMap.java:1466)
[2022-03-29T16:31:25.181Z] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[2022-03-29T16:31:25.181Z] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[2022-03-29T16:31:25.181Z] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2022-03-29T16:31:25.181Z] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[2022-03-29T16:31:25.181Z] 	at java.base/java.io.ObjectStreamClass.invokeReadObject(ObjectStreamClass.java:1175)
[2022-03-29T16:31:25.181Z] 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2478)
[2022-03-29T16:31:25.181Z] 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2344)
[2022-03-29T16:31:25.181Z] 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1803)
[2022-03-29T16:31:25.181Z] 	at java.base/java.io.ObjectInputStream.defaultReadFields(ObjectInputStream.java:2619)
[2022-03-29T16:31:25.181Z] 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2511)
[2022-03-29T16:31:25.181Z] 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2344)
[2022-03-29T16:31:25.181Z] 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1803)
[2022-03-29T16:31:25.181Z] 	at java.base/java.io.ObjectInputStream.defaultReadFields(ObjectInputStream.java:2619)
[2022-03-29T16:31:25.181Z] 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2511)
[2022-03-29T16:31:25.181Z] 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2344)
[2022-03-29T16:31:25.181Z] 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1803)
[2022-03-29T16:31:25.181Z] 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:577)
[2022-03-29T16:31:25.181Z] 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:487)
[2022-03-29T16:31:25.181Z] 	at java.rmi/sun.rmi.server.UnicastRef.unmarshalValue(UnicastRef.java:327)
[2022-03-29T16:31:25.181Z] 	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:339)
[2022-03-29T16:31:25.181Z] 	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
[2022-03-29T16:31:25.181Z] 	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
[2022-03-29T16:31:25.181Z] 	at java.base/java.security.AccessController.doPrivileged(AccessController.java:783)
[2022-03-29T16:31:25.181Z] 	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
[2022-03-29T16:31:25.181Z] 	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
[2022-03-29T16:31:25.181Z] 	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
[2022-03-29T16:31:25.181Z] 	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
[2022-03-29T16:31:25.181Z] 	at java.base/java.security.AccessController.doPrivileged(AccessController.java:717)
[2022-03-29T16:31:25.181Z] 	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
[2022-03-29T16:31:25.181Z] 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2022-03-29T16:31:25.181Z] 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2022-03-29T16:31:25.181Z] 	at java.base/java.lang.Thread.run(Thread.java:871)
[2022-03-29T16:31:25.181Z] 	at java.rmi/sun.rmi.transport.StreamRemoteCall.exceptionReceivedFromServer(StreamRemoteCall.java:303)
[2022-03-29T16:31:25.181Z] 	at java.rmi/sun.rmi.transport.StreamRemoteCall.executeCall(StreamRemoteCall.java:279)
[2022-03-29T16:31:25.181Z] 	at java.rmi/sun.rmi.server.UnicastRef.invoke(UnicastRef.java:164)
[2022-03-29T16:31:25.181Z] 	at java.rmi/java.rmi.server.RemoteObjectInvocationHandler.invokeRemoteMethod(RemoteObjectInvocationHandler.java:217)
[2022-03-29T16:31:25.181Z] 	at java.rmi/java.rmi.server.RemoteObjectInvocationHandler.invoke(RemoteObjectInvocationHandler.java:162)
[2022-03-29T16:31:25.181Z] 	at com.sun.proxy.$Proxy95.compile(Unknown Source)
[2022-03-29T16:31:25.181Z] 	at org.jetbrains.kotlin.compilerRunner.GradleKotlinCompilerWork.incrementalCompilationWithDaemon(GradleKotlinCompilerWork.kt:290)
```

- quarkus_test:
```
[2022-03-29T16:28:09.820Z] [ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.0.0-M5:test (default-test) on project quarkus-resteasy-reactive-deployment: There are test failures.
[2022-03-29T16:28:09.820Z] [ERROR] 
[2022-03-29T16:28:09.820Z] [ERROR] Please refer to /quarkus/extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/target/surefire-reports for the individual test results.
[2022-03-29T16:28:09.820Z] [ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.
[2022-03-29T16:28:09.820Z] [ERROR] There was an error in the forked process
[2022-03-29T16:28:09.820Z] [ERROR] Java heap space
[2022-03-29T16:28:09.820Z] [ERROR] org.apache.maven.surefire.booter.SurefireBooterForkException: There was an error in the forked process
[2022-03-29T16:28:09.820Z] [ERROR] Java heap space
[2022-03-29T16:28:09.820Z] [ERROR] 	at org.apache.maven.plugin.surefire.booterclient.ForkStarter.fork(ForkStarter.java:733)
[2022-03-29T16:28:09.820Z] [ERROR] 	at org.apache.maven.plugin.surefire.booterclient.ForkStarter.run(ForkStarter.java:305)
[2022-03-29T16:28:09.820Z] [ERROR] 	at org.apache.maven.plugin.surefire.booterclient.ForkStarter.run(ForkStarter.java:265)
[2022-03-29T16:28:09.820Z] [ERROR] 	at org.apache.maven.plugin.surefire.AbstractSurefireMojo.executeProvider(AbstractSurefireMojo.java:1314)
[2022-03-29T16:28:09.820Z] [ERROR] 	at org.apache.maven.plugin.surefire.AbstractSurefireMojo.executeAfterPreconditionsChecked(AbstractSurefireMojo.java:1159)
[2022-03-29T16:28:09.820Z] [ERROR] 	at org.apache.maven.plugin.surefire.AbstractSurefireMojo.execute(AbstractSurefireMojo.java:932)
[2022-03-29T16:28:09.820Z] [ERROR] 	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:137)
[2022-03-29T16:28:09.820Z] [ERROR] 	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:210)
[2022-03-29T16:28:09.820Z] [ERROR] 	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:156)
[2022-03-29T16:28:09.820Z] [ERROR] 	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:148)
[2022-03-29T16:28:09.820Z] [ERROR] 	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:117)
[2022-03-29T16:28:09.820Z] [ERROR] 	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:81)
[2022-03-29T16:28:09.820Z] [ERROR] 	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:56)
[2022-03-29T16:28:09.820Z] [ERROR] 	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
[2022-03-29T16:28:09.820Z] [ERROR] 	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:305)
[2022-03-29T16:28:09.820Z] [ERROR] 	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:192)
[2022-03-29T16:28:09.820Z] [ERROR] 	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:105)
[2022-03-29T16:28:09.820Z] [ERROR] 	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:972)
[2022-03-29T16:28:09.820Z] [ERROR] 	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:293)
[2022-03-29T16:28:09.820Z] [ERROR] 	at org.apache.maven.cli.MavenCli.main(MavenCli.java:196)
[2022-03-29T16:28:09.820Z] [ERROR] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[2022-03-29T16:28:09.820Z] [ERROR] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[2022-03-29T16:28:09.820Z] [ERROR] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2022-03-29T16:28:09.820Z] [ERROR] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[2022-03-29T16:28:09.820Z] [ERROR] 	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:282)
[2022-03-29T16:28:09.820Z] [ERROR] 	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:225)
[2022-03-29T16:28:09.820Z] [ERROR] 	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:406)
[2022-03-29T16:28:09.820Z] [ERROR] 	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:347)
[2022-03-29T16:28:09.820Z] [ERROR] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[2022-03-29T16:28:09.820Z] [ERROR] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[2022-03-29T16:28:09.820Z] [ERROR] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2022-03-29T16:28:09.820Z] [ERROR] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[2022-03-29T16:28:09.820Z] [ERROR] 	at org.apache.maven.wrapper.BootstrapMainStarter.start(BootstrapMainStarter.java:47)
[2022-03-29T16:28:09.820Z] [ERROR] 	at org.apache.maven.wrapper.WrapperExecutor.execute(WrapperExecutor.java:156)
[2022-03-29T16:28:09.820Z] [ERROR] 	at org.apache.maven.wrapper.MavenWrapperMain.main(MavenWrapperMain.java:72)
[2022-03-29T16:28:09.820Z] [ERROR] 
[2022-03-29T16:28:09.820Z] [ERROR] -> [Help 1]
[2022-03-29T16:28:09.820Z] [ERROR] 
[2022-03-29T16:28:09.820Z] [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[2022-03-29T16:28:09.820Z] [ERROR] Re-run Maven using the -X switch to enable full debug logging.
[2022-03-29T16:28:09.820Z] [ERROR] 
[2022-03-29T16:28:09.820Z] [ERROR] For more information about the errors and possible solutions, please read the following articles:
[2022-03-29T16:28:09.820Z] [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[2022-03-29T16:28:09.820Z] [ERROR] 
[2022-03-29T16:28:09.820Z] [ERROR] After correcting the problems, you can resume the build with the command
[2022-03-29T16:28:09.820Z] [ERROR]   mvn <args> -rf :quarkus-resteasy-reactive-deployment
[2022-03-29T16:28:34.704Z] 
[2022-03-29T16:28:34.704Z] quarkus_test_0_FAILED
```

**JDK 11 hotspot:**
- quarkus_test: https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/4169/
```
[2022-03-29T18:18:24.829Z] [ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.0.0-M5:test (default-test) on project quarkus-opentelemetry-deployment: There are test failures.
[2022-03-29T18:18:24.829Z] [ERROR] 
[2022-03-29T18:18:24.829Z] [ERROR] Please refer to /quarkus/extensions/opentelemetry/opentelemetry/deployment/target/surefire-reports for the individual test results.
[2022-03-29T18:18:24.829Z] [ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.
[2022-03-29T18:18:24.829Z] [ERROR] -> [Help 1]
[2022-03-29T18:18:24.829Z] [ERROR] 
[2022-03-29T18:18:24.829Z] [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[2022-03-29T18:18:24.829Z] [ERROR] Re-run Maven using the -X switch to enable full debug logging.
[2022-03-29T18:18:24.829Z] [ERROR] 
[2022-03-29T18:18:24.829Z] [ERROR] For more information about the errors and possible solutions, please read the following articles:
[2022-03-29T18:18:24.829Z] [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[2022-03-29T18:18:24.829Z] [ERROR] 
[2022-03-29T18:18:24.829Z] [ERROR] After correcting the problems, you can resume the build with the command
[2022-03-29T18:18:24.829Z] [ERROR]   mvn <args> -rf :quarkus-opentelemetry-deployment
[2022-03-29T18:18:42.270Z] 
[2022-03-29T18:18:42.270Z] quarkus_test_0_FAILED
```
- spring_test: https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/4170/

```
00:04:26.799  =JAVA VERSION OUTPUT END=
00:04:26.799  Compile and execute spring-test
00:04:26.799  Building Spring  using gradle
00:04:26.799  Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF8 -Djava.security.egd=file:/dev/./urandom
00:08:05.092  Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF8 -Djava.security.egd=file:/dev/./urandom
00:08:10.839  Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF8 -Djava.security.egd=file:/dev/./urandom
00:08:50.592  Compilation with Kotlin compile daemon was not successful
00:08:50.592  java.lang.NullPointerException
00:08:50.592  	at org.jetbrains.kotlin.incremental.IncrementalModuleEntry.hashCode(IncrementalModuleInfo.kt)
00:08:50.592  	at java.base/java.util.HashMap.hash(HashMap.java:340)
00:08:50.592  	at java.base/java.util.HashMap.put(HashMap.java:608)
00:08:50.592  	at java.base/java.util.HashSet.readObject(HashSet.java:343)
00:08:50.592  	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
00:08:50.592  	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
00:08:50.592  	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
00:08:50.592  	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
00:08:50.592  	at java.base/java.io.ObjectStreamClass.invokeReadObject(ObjectStreamClass.java:1175)
00:08:50.592  	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2351)
00:08:50.592  	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2222)
...
```

--- a/external/quarkus/playlist.xml
+++ b/external/quarkus/playlist.xml
@@ -20,7 +20,7 @@
 		$(TEST_ROOT)$(D)external$(D)external.sh --clean --tag "${DOCKERIMAGE_TAG}" --dir quarkus
 		</command>
 		<levels>
-			<level>extended</level>
+			<level>sanity</level>
 		</levels>
 		<groups>
 			<group>external</group>
@@ -41,7 +41,7 @@
 		$(TEST_ROOT)$(D)external$(D)external.sh --clean --tag "${DOCKERIMAGE_TAG}" --dir quarkus
 		</command>
 		<levels>
-			<level>extended</level>
+			<level>sanity</level>
 		</levels>
 		<groups>
 			<group>external</group>

--- a/external/spring/playlist.xml
+++ b/external/spring/playlist.xml
@@ -26,7 +26,7 @@
 			$(TEST_ROOT)$(D)external$(D)external.sh --clean --tag "${DOCKERIMAGE_TAG}" --dir spring
 		</command>
 		<levels>
-			<level>extended</level>
+			<level>sanity</level>
 		</levels>
 		<groups>
 			<group>external</group>

